config/v1/tests/nodes.config.openshift.io/MinimumKubeletVersion.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Node"
+crdName: nodes.config.openshift.io
+tests:
+  onCreate:
```

Comment:
Should add some tests to test the validations you've added.

For example, making sure that the empty string is accepted, going too long, invalid formats etc